packages/botonic-react/tests/components/render-markdown.test.jsx

```diff
@@ -0,0 +1,170 @@
+import { renderMarkdown } from '../../src/components/markdown'
+
+describe('Using renderMarkdown', () => {
+  // Markdown renderer adding an extra blank space after each render
```

Comment:
where is the thing about "extra blank space"?

Reply:
I added a clarification, so the result of the renderer in the test was:
`<p>Text<p> ` (note extra blank space) instead of `<p>Text</p>`, so I'm trimming this last space for the tests.